lab/src/Microsoft.ServiceModel.Syndication/src/Atom10FeedFormatter.cs

```diff
@@ -409,175 +508,186 @@ internal void WriteContentTo(XmlWriter writer, string elementName, SyndicationCo
             }
         }
 
-        internal void WriteElement(XmlWriter writer, string elementName, string value)
+        internal async Task WriteElementAsync(XmlWriterWrapper writer, string elementName, string value)
         {
             if (value != null)
             {
-                writer.WriteElementString(elementName, Atom10Constants.Atom10Namespace, value);
+                await writer.WriteElementStringAsync(elementName, Atom10Constants.Atom10Namespace, value);
             }
         }
 
-        internal void WriteFeedAuthorsTo(XmlWriter writer, Collection<SyndicationPerson> authors)
+        internal async Task WriteFeedAuthorsToAsync(XmlWriter writer, Collection<SyndicationPerson> authors)
         {
             for (int i = 0; i < authors.Count; ++i)
             {
                 SyndicationPerson p = authors[i];
-                WritePersonTo(writer, p, Atom10Constants.AuthorTag);
+                await WritePersonToAsync(XmlWriterWrapper.CreateFromWriter(writer), p, Atom10Constants.AuthorTag);
             }
         }
 
-        internal void WriteFeedContributorsTo(XmlWriter writer, Collection<SyndicationPerson> contributors)
+        internal async Task WriteFeedContributorsToAsync(XmlWriter writer, Collection<SyndicationPerson> contributors)
         {
             for (int i = 0; i < contributors.Count; ++i)
             {
                 SyndicationPerson p = contributors[i];
-                WritePersonTo(writer, p, Atom10Constants.ContributorTag);
+                await WritePersonToAsync(XmlWriterWrapper.CreateFromWriter(writer), p, Atom10Constants.ContributorTag);
             }
         }
 
-        internal void WriteFeedLastUpdatedTimeTo(XmlWriter writer, DateTimeOffset lastUpdatedTime, bool isRequired)
+        internal async Task WriteFeedLastUpdatedTimeToAsync(XmlWriterWrapper writer, DateTimeOffset lastUpdatedTime, bool isRequired)
         {
```

Comment:
Remove async from method. Return Task directly from WriteElementAsync. Outside second if block, return Task.CompletedTask. Add blank line after closing braces of if blocks.